metadata_input_netlink.c

```diff
@@ -130,14 +139,6 @@ static uint8_t md_input_netlink_parse_conn_event(struct md_input_netlink *min,
         return RETVAL_FAILURE;
     }
 
-    //We need to update the value in case of a connection event update, since it
-    //is a string
-    //TODO: Implement a more elegant technique if we get more cases like this
-    if (mce->event_param == CONN_EVENT_META_UPDATE && !mce->event_value_str) {
-        META_PRINT_SYSLOG(min->parent, LOG_ERR, "Missing event value for connection update\n");
-        return RETVAL_FAILURE;
-    }
```

Comment:
I guess removing this conditional is safe, since dlb will not export bogus updates? Or if we export bogus updates, we have bigger problems :)